build-logic/src/main/kotlin/CompilerOptions.kt

```diff
@@ -56,7 +63,7 @@ fun Project.configureJavaAndKotlinCompilers(treatWarningsAsErrors: Boolean = fal
 private fun Project.treatWarningsAsErrors() {
   tasks.withType(KotlinCompile::class.java) {
     kotlinOptions {
-      allWarningsAsErrors = true
+      //allWarningsAsErrors = true
```

Comment:
On purpose?

Reply:
Yes, sadly `languageVersion = 1.5` outputs a warning that we can't selectively disable. I'll add a comment